### PR TITLE
Update richtext.lua to add richtext.remove()

### PR DESCRIPTION
**Pull request by 8bitskull**
Added function richtext.remove(). It's trivial to loop over the words table and do it yourself, but it looks a bit cluttered when you are attempting to delete multiple rich text tables.

### CODE REVIEW

File: richtext/richtext.lua [hunk 1]
```diff
@@ -634,5 +634,15 @@ function M.characters(word)
 	return chars
 end
 
+---Removes the gui nodes created by rich text
+function M.remove(words)
+	assert(words)
+
+	local num = table.getn(words)
```
Comment (britzl) [inline richtext/richtext.lua:641]: table.getn() was deprecated in 5.1. The length operator (#) is the recommended way to get the length of a table:

https://www.lua.org/manual/5.1/manual.html#7.2

**Review by britzl — CHANGES_REQUESTED** [review]
Makes sense to add richtext.remove(). Please see comment about table.getn().

Please also update README.md with the new function!

File: richtext/richtext.lua (hunk 1 above)
Comment (8bitskull) [reply richtext/richtext.lua]: Updated to replace table.getn() with #.
Added documentation.

**Review by britzl — APPROVED** [review]
Thanks!

**Pull request merged**